AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Sync/InitialSync/InitialSyncOperation.swift

```diff
@@ -54,7 +54,19 @@ final class InitialSyncOperation: AsynchronousOperation {
         }
 
         let lastSyncMetadata = getLastSyncMetadata()
-        lastSyncTime = lastSyncMetadata?.lastSync
+        if let lastSync = lastSyncMetadata?.lastSync {
+            let lastSyncDate = Date(timeIntervalSince1970: TimeInterval(lastSync) / 1_000)
```

Comment:
It'd be convenient to extend @drochetti's `TimeInterval` extension that provides unit-based constructors with a `milliseconds` so you could just do `lastSyncDate = Date(timeIntervalSince1970: TimeInterval.milliseconds(lastSync))`

Reply:
Hmmm yes, this would be cool.  This is not currently in `master`, so adding a todo, and